src/core/server/saved_objects/migrations/integration_tests/cluster_routing_allocation_disabled.test.ts

```diff
@@ -149,7 +150,8 @@ describe('unsupported_cluster_routing_allocation', () => {
     await root.setup();
 
     await expect(root.start()).rejects.toThrowError(
-      /Unable to complete saved object migrations for the \[\.kibana.*\] index: The elasticsearch cluster has cluster routing allocation incorrectly set for migrations to continue\. To proceed, please remove the cluster routing allocation settings with PUT \/_cluster\/settings {"transient": {"cluster\.routing\.allocation\.enable": null}, "persistent": {"cluster\.routing\.allocation\.enable": null}}/
+      // eslint-disable-next-line prettier/prettier
```

Comment:
Open to suggestions on how to escape the chars to override prettier formatting!

Reply:
I think this should work, but as mentioned above, I'm not sure you need to disable prettier anyway.

```js
/Unable to complete saved object migrations for the \[\.kibana\] index: \[unsupported_cluster_routing_allocation\] The elasticsearch cluster has cluster routing allocation incorrectly set for migrations to continue\. To proceed, please remove the cluster routing allocation settings with PUT \/_cluster\/settings {\"transient\": {\"cluster\.routing\.allocation\.enable\": null}, \"persistent\": {\"cluster\.routing\.allocation\.enable\": null}}\. Refer to https:\/\/www.elastic.co\/guide\/en\/kibana\/master\/resolve-migrations-failures.html#routing-allocation-disabled for more information on how to resolve the issue\./
```